netcontrold/tests/lib/test_prelim.py

```diff
@@ -0,0 +1,66 @@
+#
+#  Copyright (c) 2020 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from unittest import TestCase
+
+from netcontrold.lib import dataif
+from netcontrold.lib.error import ObjConsistencyExc
+
+class TestDataif_add_rxq(TestCase):
+
+	def setUp(self):
+		self.port1= dataif.Port("port1")
+
+
+	def test_port_add_rxq_1(self):
+		self.rxq1=self.port1.add_rxq(1)
+		self.assertEqual(self.port1.find_rxq_by_id(1),self.rxq1)
+
+	# def test_port_add_rxq_2(self):
+	# self.rxq2=self.port1.add_rxq(2)
+	# 	self.assertRaises(ObjConsistencyExc,self.port1.add_rxq(2)) 
```

Comment:
Well, expectation should be that, new rxq should not be added when there is a rxq of same id already created.
Image below pmd/rxq mapping:
pmd 1:
  port 1 - rxq 0
pmd 2:
  port 1 - rxq 0 << impossible >>
pmd 3:
  port 1 - rxq 1 << possible >>

We should raise exception for this impossible scenario, so you did right there. So, go ahead for:
(1) fixing lib/dataif.py to raise exception where there is queue of same ID exists for the port.
(2) uncomment this test

Please use separate commits (refer my PR #47 for how I stage commits independently where ever needed).